apps/templates/content_templates.py

```diff
@@ -335,7 +349,8 @@ def get_item_from_template(template):
     """
     item = template.get('data', {})
     item[ITEM_STATE] = CONTENT_STATE.SUBMITTED
-    item['task'] = {'desk': template.get('template_desk'), 'stage': template.get('template_stage')}
+    if template.get('schedule_desk'):
```

Comment:
The problem here is that the code runs in a celery task so no notification will be sent to the user if we raise an error.